<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LineEdits</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="ColorLineEdit" name="lineEdit">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="ColorLineEdit" name="lineEdit_2">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="ColorLineEdit" name="lineEdit_3">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="ColorLineEdit" name="lineEdit_4">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="ColorLineEdit" name="lineEdit_5">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="ColorLineEdit" name="lineEdit_6">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="ColorLineEdit" name="lineEdit_7">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="ColorLineEdit" name="lineEdit_8">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="ColorLineEdit" name="lineEdit_9">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="ColorLineEdit" name="lineEdit_10">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="ColorLineEdit" name="lineEdit_11">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="ColorLineEdit" name="lineEdit_12">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Tab Direction</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>164</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>&amp;Lines (F2)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>&amp;Columns (F3)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorLineEdit</class>
   <extends>QLineEdit</extends>
   <header>colorlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
